src/plasoscaffolder/bll/mappings/mapping_helper.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""Helper methods for mapping"""
+import os
+
+from jinja2 import Environment
+from jinja2 import FileSystemLoader
+
+
+def _get_template_path() -> os.path:
+  """
+  the path to the template file
+  :param file_name: the name of the file
+  :return: the template file path
+  """
+  path = os.path.dirname(os.path.abspath(__file__))
+  head, tail = os.path.split(path)
+  return os.path.join(head, 'templates')
```

Comment:
'template', please stick with 1 style of string quotes either " or '

For context plaso uses ' for single quoted string and """ for multi line strings
https://github.com/log2timeline/plaso/wiki/Style-guide#strings